endpoints/users/controller.py

```diff
@@ -58,15 +57,19 @@ def post(self):
             return marshal(user, user_list_fields)
 
         except Exception as error:
-            response.errorResponse(str(error))
-            return response.__dict__
+            return {'message': 'Something went wrong', 'timestamp': round(time.time())}, 500
```

Comment:
What about you use `datetime.datetime.now().isoformat()` instead?

Reply:
Personal preference for timestamps over ISO Dates

Reply:
This is not a timestamp though. This is the epoch time.

Reply:
Ok, changed to datetime.datetime.now().isoformat()